src/utils/routes/Routes.tsx

```diff
@@ -36,21 +36,13 @@ export const InstitutionAdminRoute: FC<LoggedInRouteProps> = ({ component, ...re
   return <PrivateRoute {...rest} component={component} isAuthorized={user.isInstitutionAdmin} />;
 };
 
-const isValidInstitutionForInstitutionAdmin = (customerUri: string) => {
-  const customerId = customerUri.split('/').pop();
-  return !!customerId && window.location.pathname.includes(customerId);
-};
-
 export const EditInstitutionRoute: FC<LoggedInRouteProps> = ({ component, ...rest }) => {
   const user = useSelector((store: RootStore) => store.user);
+  const encodedCustomerId = user.customerId ? encodeURIComponent(user.customerId) : '';
+  const institutionAdminCanEditCustomer =
+    user.isInstitutionAdmin && encodedCustomerId && window.location.search.includes(encodedCustomerId);
```

Comment:
vil ikke `encodedCustomerId` alltid være true? i og med at den defaulter til `''`? 

ser ut som `institutionAdminCanEditCustomer = user.isInstitutionAdmin` fordi resten er jo alltid true?

Reply:
usikker på om jeg skjønner hva du mener, for tom streng blir false?
![bilde](https://user-images.githubusercontent.com/6313468/94120962-0a7d2080-fe51-11ea-94eb-ad4fcedec5ff.png)

Reply:
ah, tenkte at tom streng var falsy 🙃  

men uansett: blir ikke `window.location.search.includes("")` alltid true?

Reply:
jo, men `encodedCustomerId` vil bli false før man kommer dit

Reply:
så hva er poenget med `window.location.search.includes(encodedCustomerId)`?

Reply:
hvis man er Institution Admin skal man bare få lov til å åpne egen Customer Institution
så vi må først sjekke at brukeren har en customerId, og hvis man har det må vi sjekke at det er denne CustomerInstitution man prøver å åpne.
uten sjekken for dette vil institution admin for NTNU kunne endre på Unit sin Customer

Reply:
egentlig er dette noe som backend skal styre såklart, så kanskje er det unødvendig på sikt, men skader ikke uansett tenker jeg

Reply:
ahh, ok 👌